database/system_model.js

```diff
@@ -0,0 +1,178 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/systemreqs', { useNewUrlParser: true });
+const autoIncrement = require('mongoose-auto-increment');
```

Comment:
Since `mongoose-auto-increment` is an npm package that doesn't depend on the `mongoose.connect` call running first, I think it makes sense to group lines 1 and 3 separately from that call. I assume that line 2 must run before line 5; otherwise, the same comment applies to line 5.

Reply:
Got it!